modules/nf-neuro/segmentation/synthseg/tests/main.nf.test.snap

```diff
@@ -0,0 +1,67 @@
+{
+    "segmentation - synthseg - options": {
+        "content": [
+            "test__mask_wm.nii.gz:md5:header,a0430ca63ccc3bb2b925694cb7568388,data,95946cf22f6adf539b65c72b49eb3b32",
+            "test__mask_gm.nii.gz:md5:header,a0430ca63ccc3bb2b925694cb7568388,data,a54f8c03a17a16d9271534c387434720",
+            "test__mask_csf.nii.gz:md5:header,a0430ca63ccc3bb2b925694cb7568388,data,3895c471bc8b06446a8adbba9fe235f7",
+            "test__parc.nii.gz:md5:header,a0430ca63ccc3bb2b925694cb7568388,data,cb9a481316ae64dceb229879bd0c6619",
+            [
+                
```

Comment:
You don't have an output resampled image. While this is correct behavior, look that it is because input data is in 1x1x1mm. Also, for the sake of covering all bases, I would use an input in another resolution, so we get the resampled image here.

If you need data uploaded, tell me and I push it for you !

Reply:
Used processing/mni_masked_2x2x2. If importing a third dataset is somewhat inefficient, we'll look to add some into freesurfer.

Reply:
Then shouldn't the resampled volume get outputed here ?

Reply:
Yes! Like you said. Needed a "not 1x1x1" test data.